### ISSUE TITLE
Create Settings Page

### ISSUE THREAD

**Issue by marceloschmidt** [RocketChat/Rocket.Chat#222]
- [ ] Configure external services authentication
- [x] SMTP settings
- [ ] Disable registration #224 
- [ ] Disable e-mail verification
- [ ] LDAP options
- [ ] Analytic settings (set ID or disable)

*Message Editing/Deleting*
  - [ ] Allow/Disallow message editing
  - [ ] Allow/Disallow message deleting
  - [ ] Show/Hide that a message has been edited/deleted
  - [ ] Keep history of edited/deleted messages

- [ ] Automatic expire channels

**Comment by sampaiodiego** [comment]
I think we may define what kind of options we must have here, like:

* Options to configure external services authentication
* External e-mail server
* Disable register (for closed teams)

@marceloschmidt can you edit the issue with the "accepted" options?

**Comment by sampaiodiego** [comment]
We should add an option to change behavior of message editing/deleting..

It could be enable or disable the feature of editing/deleting messages or keep it's history maybe.

**Comment by nevson** [comment]
- [ ] Analytic settings
  - i.e. allow adding new analytic trackers or disable them completely, see https://github.com/RocketChat/Rocket.Chat/issues/217

**Comment by haosdent** [comment]
+1

**Comment by stevenolen** [comment]
is this implemented now? I've been meaning to get started with rocketchat but was waiting on this one! :+1:

**Comment by stevenolen** [comment]
...for anyone else visiting, it's referenced in another issue: `/settings`. :tada:

**Comment by marceloschmidt** [comment]
And in case you've missed it, you gotta be admin. So if you need to access /settings on your own instance, set admin: true under your user in mongodb.

**Comment by Wloa** [comment]
@Marcelo, there is no admin field in users collection. 

On 29.07.2015 18:06, Marcelo Schmidt wrote: 

> And in case you've missed it, you gotta be admin. So if you need to access /settings on your own instance, set admin: true under your user in mongodb. 
> 
> --
> Reply to this email directly or view it on GitHub [1].
 

Links:
------
[1]
https://github.com/RocketChat/Rocket.Chat/issues/222#issuecomment-126110578

**Comment by sampaiodiego** [comment]
@Wloa create the field with value 'true'

**Comment by marceloschmidt** [comment]
@Wloa Exactly as @sampaiodiego said, just create it, like so: 
```json
{
  "_id": "jrnR99viqXELp4XYn",
  "username": "marcelo.rocket.team",
  ...
  "admin": true
}
```

**Comment by engelgabriel** [comment]
@marceloschmidt can you create a migration turning the oldest user into admin on the next release update?

**Comment by marceloschmidt** [comment]
Done
